cognite_toolkit/_cdf_tk/templates/_templates.py

```diff
@@ -610,6 +601,22 @@ def validate(content: str, destination: Path, source_path: Path, modules_by_vari
             print(f"    Available resources are: {', '.join(LOADER_BY_FOLDER_NAME.keys())}")
             return
 
+        if isinstance(parsed, dict):
+            parsed = [parsed]
+        for item in parsed:
+            if not check_yaml_semantics(
+                parsed=item,
+                filepath_src=source_path,
+                filepath_build=destination,
+            ):
+                print(
+                    f"  [bold yellow]WARNING:[/] In module {source_path.parent.parent.name!r}, the resource {destination.parent.name!r} is not semantically correct."
```

Comment:
A bit hard to know what the actual file that is wrong.
```suggestion
                    f"  [bold yellow]WARNING:[/] In module {source_path.parent.parent.name!r}, the resource {destination.parent.name!r}/{destination.name} is not semantically correct."
```